src/main/java/javax/jmdns/JmmDNS.java

```diff
@@ -1,3 +1,16 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 /**
  *
  */
```

Comment:
Here's an old empty header.

Reply:
Yep it looks like the plugin only fixes real license headers and 

```
\\\ header
```

And double star is not a license on single star`

Comment not a license...
```
/**
*
```

Reply:
Fixed